.github/workflows/jobs-integration-test.yml

```
@@ -0,0 +1,59 @@
+name: Integration test
+on:
+  workflow_call:
+    inputs:
+      environment_name:
+        type: string
+        required: true
+        description: The name of the environment.
+      project_prefix_name:
+        type: string
+        required: true
+        description: The prefix name of the project
+    secrets:
+      AZURE_CREDENTIALS:
+        required: true
+    
+env:
+  Bot1Url: "https://byk-dev-aks-ingress.westeurope.cloudapp.azure.com/mock-bot"
+  CentOpsUrl: "https://byk-dev-aks-ingress.westeurope.cloudapp.azure.com/centops"
```

It might be worth building up these URLs with the project and environment names as the might help any future devs to validate changes in other environments.

```suggestion
  Bot1Url: "https://${{ inputs.project_prefix_name }}-${{ inputs.environment_name }}-aks-ingress.westeurope.cloudapp.azure.com/mock-bot"
  CentOpsUrl: "https://${{ inputs.project_prefix_name }}-${{ inputs.environment_name }}-aks-ingress.westeurope.cloudapp.azure.com/centops"
```